src/overlays/actors/ovl_En_Ds/z_en_ds.c

```diff
@@ -21,43 +44,250 @@ const ActorInit En_Ds_InitVars = {
     (ActorFunc)EnDs_Update,
     (ActorFunc)EnDs_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ds/EnDs_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ds/EnDs_Destroy.s")
+extern UNK_TYPE D_06004768;
```

Comment:
This type of this is Skeleton
```suggestion
extern Skeleton D_06004768;
```